src/modules/Nfs.ycp

```diff
@@ -352,6 +354,89 @@
 	return true;
     }
 
+    typedef string vertex_t;
+
+    vertex_t VertexFromEntry(map<string, any> e) {
+      return String::PadZeros(tostring(e["i"]:0), 6) + e["file"]:"";
+    }
+
+    string PathFromVertex(vertex_t v) {
+      return substring(v, 6);
+    }
+
+    /*
+     * DependsOn(a, b) is true if a must be mounted before b.
+     * That is, "/any" DependsOn "/", "/foo/bar" DependsOn "/foo",
+     * but "/foof" not DependsOn "/foo"
+     */
+    boolean DependsOn(vertex_t a, vertex_t b) {
+      a = PathFromVertex(a);
+      b = PathFromVertex(b);
+
+      if (a == b) return false;
+
+      boolean StartsWith(string s, string head) {
```

Comment:
Please use existing method, do not reinvent wheel - https://github.com/yast/yast-yast2/blob/Code-11-SP4/library/types/src/String.ycp#L1248